Dockerfile-test

```diff
@@ -0,0 +1,2 @@
+FROM fxa-custom-server:build
```

Comment:
The reason it's trying to pull from docker hub is the missing `s` here, should be `fxa-customs-server`